File: src/sdk-reference/cpp/1/document/delete/snippets/delete.cpp
try {
  std::string id = response = kuzzle->document->delete_(
    "nyc-open-data",
    "yellow-taxi",
    "some-id"
  );

  std::cout << "Document " << id << " successfully deleted" << std::endl;
} catch (kuzzleio::KuzzleException& e) {
  std::cerr << e.what() << std::endl;
}

File: src/sdk-reference/java/1/document/delete/snippets/delete.java
try {
  String id = kuzzle.getDocument().delete(
    "nyc-open-data",
    "yellow-taxi",
    "some-id");

  System.out.println("Document " + id + " successfully deleted");
} catch (KuzzleException e) {
  System.err.println(e.getMessage());
}
